packages/fx-core/tests/component/driver/arm/utils.test.ts

```diff
@@ -304,24 +307,25 @@ describe("arm deploy error handle test", () => {
     }
   });
 
-  it("deployTemplate throw none FxError", async () => {
+  it("deployTemplate throw unresolved env variable FxError", async () => {
     const deployArgs = {
       subscriptionId: "00000000-0000-0000-0000-000000000000",
       resourceGroupName: "mock-group",
       bicepCliVersion: "",
       templates: [],
     } as any;
-    mockedDriverContext.createProgressBar = () => {};
     const impl = new ArmDeployImpl(deployArgs, mockedDriverContext);
-    sandbox.stub(impl, "getDeployParameters").throws(new Error("compile error"));
+    sandbox.stub(fs, "readFile").resolves("{}" as any);
+    sandbox.stub(common, "getEnvironmentVariables").returns(["env1", "env2", "env3"]);
```

Comment:
Can you add some non existence envs to parameter in line 323 and remove this mock? `getEnvironmentVariables` should be tested in the UT.

Reply:
Updated. Thanks.